docs/testing.md

```diff
@@ -1,17 +1,40 @@
 # Testing
 
+[Official Go test reference](https://pkg.go.dev/cmd/go#hdr-Testing_flags)
+
 ## Backend
 
 Backend testing includes both unit testing and integration testing. Because manual testing is less efficient, all new features or code changes need defence from automated testing. Without it, no one knows when things will break.
 
-All the Go integration tests is at [tests directory](https://github.com/bytebase/bytebase/tree/main/tests). You can use the following command to run all tests or one test.
+All the Go integration tests is at [tests directory](https://github.com/bytebase/bytebase/tree/main/tests).
+
+### Prerequisites
+
+We embed MySQL binaries for testing. Run following command to download MySQL distributions first.
+
+```shell
+# Run this from the repo root
+go generate -tags mysql ./...
+```
+
+### Run tests
+
+#### Run all backend tests
+
+```shell
+alias ta="go test --tags=mysql -v ./..."
```

Comment:
If we want to run tests recursively, then we can't supply -run

Reply:
I ran "t ./..." for this.

Reply:
![CleanShot 2022-11-29 at 12 29 58](https://user-images.githubusercontent.com/230323/204439204-42678239-e1d1-4a3c-bf42-995a340d9b0d.png)

This doesn't work on my zsh terminal if running from the root using go 1.9. So my understanding is if we specify -run, then it will not look for a recursive directory.

Reply:
-run receive a regex, and match it recursively, you can use t "Test" ./... to verify this logic.

Reply:
Then with the -run option, we need to use t "" ./... right?

Reply:
Yeah, it works for me.